tiledb/sm/group/group.cc

```diff
@@ -67,7 +73,8 @@ Group::Group(const URI& group_uri, StorageManager* storage_manager)
     , query_type_(QueryType::READ)
     , timestamp_start_(0)
     , timestamp_end_(UINT64_MAX)
-    , encryption_key_(tdb::make_shared<EncryptionKey>(HERE())) {
+    , encryption_key_(tdb::make_shared<EncryptionKey>(HERE()))
+    , resources_(resources) {
```

Comment:
Another note for the future: Looking at this again, please not that this member variable `resources_` should probably be declared first in the header and initialized first in the initializer list. It should be in that position so that any other member variable that might need a resource for its own initialization has it available.

No need to do anything for this PR.